rel-eng/fedora-pulp.repo

```diff
@@ -63,3 +31,11 @@ baseurl=https://repos.fedorapeople.org/repos/pulp/pulp/testing/2.6/fedora-$relea
 enabled=0
 skip_if_unavailable=1
 gpgcheck=0
+
+# Version 2.7 Testing Builds
+[pulp-2.7-testing]
+name=Pulp 2.7 Testing Builds
+baseurl=https://repos.fedorapeople.org/repos/pulp/pulp/testing/automation/2.7/dev/fedora-$releasever/$basearch/
+enabled=0
+skip_if_unavailable=1
+gpgcheck=0
```

Comment:
(adding @barnabycourt to cc)

This url points to an automation repo which isn't signed. We'd need to start publishing and signing alphas if we wanted to do this.

Would calling the repo something like `pulp-2.7-nightly` be more accurate? I just need the repo definition available somewhere so I can install w/ Vagrant from rpm. A nightly would work better for my purposes than a hand-tagged alpha since it would be fresher.